submissions/kt-std/memory-pair-game/js/script.js

```diff
@@ -0,0 +1,145 @@
+const container = document.querySelector(".container"),
+	numCards = 9,
+	cardsData = ((assetsPath) => {
+		const arr = [];
+		for (let i = 1; i <= numCards; i++) {
```

Comment:
let's try to use `Array.prototype.map` here

Reply:
I’m afraid I’m a little confused how to do this as this function just creates a cardData array that contains "generated" names and pathes.

Reply:
Would it be okay if I create an array of empty slots with dedicated length and then apply map to it to iterate through all its elements and return the new one?

Reply:
I've just updated the code with the map method. I do hope it is better now.